apps/functions-subscription/src/adapters/azure/servicebus/__tests__/mocks.ts

```diff
@@ -1,7 +1,8 @@
-import { vi } from 'vitest';
+import { mockDeep } from 'vitest-mock-extended';
+import { ServiceBusManagementClient } from '@azure/arm-servicebus';
+import { ServiceBusSender } from '@azure/service-bus';
 
-export const makeServiceBusMock = () => {
-  return {
-    sendMessages: vi.fn(),
-  };
-};
+export const makeServiceBusMocks = () => ({
+  sender: mockDeep<ServiceBusSender>(),
```

Comment:
**note**: here I just remove the old way to crate a mock and replaced with the vitest-mock-extended functionality